Week-6/6.5/assignment6.5.js

```diff
@@ -0,0 +1,22 @@
+function checkDiff(arr, diff) {
+    let num1, num2;
+
+    arr.forEach(element => {
```

Comment:
@mr-rego looks good to me. But a better way to solve in O(n), please go through that for your reference

https://www.geeksforgeeks.org/find-a-pair-with-the-given-difference/